app/views/publications/monthly_statistics/show.html.erb

```diff
@@ -2,11 +2,27 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-xl">Statistics</span>
-    <h1 class="govuk-heading-xl">
+    <span class="govuk-caption-l">Statistics</span>
+    <h1 class="govuk-heading-l">
       Initial teacher training applications for courses starting in the <%= @presenter.next_cycle_name %> academic year
     </h1>
+  </div>
+</div>
+<div class="govuk-grid-row">
+  <div class="metadata-wrapper">
+    <div class="govuk-grid-column-two-thirds metadata-column">
+      <dl class="gem-c-metadata">
+        <dt class="gem-c-metadata__term">Published:</dt>
+        <dd class="gem-c-metadata__definition"><%= @presenter.publication_date.to_fs(:govuk_date) %></dd>
+        <dt class="gem-c-metadata__term">Next update:</dt>
+        <dd class="gem-c-metadata__definition"><%= @presenter.next_publication_date.to_fs(:govuk_date) %></dd>
```

Comment:
The style on the GOVUK versions of these pages is not to use colons for these dates (although they do use them for 'From:' with the org name... 🤷 ).

```suggestion
        <dt class="gem-c-metadata__term">Published</dt>
        <dd class="gem-c-metadata__definition"><%= @presenter.publication_date.to_fs(:govuk_date) %></dd>
        <dt class="gem-c-metadata__term">Next update</dt>
        <dd class="gem-c-metadata__definition"><%= @presenter.next_publication_date.to_fs(:govuk_date) %></dd>
```